shanghai/connection.py

```diff
@@ -38,7 +39,7 @@ def close(self):
             while not reader.at_eof():
                 line = await reader.readline()
                 line = line.strip()
-                current_logger.debug(">", line)
+                current_logger.info(">", line)
```

Comment:
I thought about this a little.

Imo, we should revert this back to debug and add a level lower than debug (name suggestions welcome) for less important messages. 
Regarding messages, we should most likely log messages that are passed to some event handler in a higher level.